cluster/charts/rook-ceph/templates/psp.yaml

```diff
@@ -14,7 +14,7 @@
 apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: 00-rook-ceph-operator
+  name: rook-privileged
```

Comment:
We should name this `00-rook-privileged`, otherwise installs can grab the default PSP, notably when other features like placement affinities are used.